packages/react/src/components/DetailsList/DetailsList.base.tsx

```diff
@@ -1385,11 +1436,11 @@ export function buildColumns(
   return columns;
 }
 
-function getPaddedWidth(column: IColumn, isFirst: boolean, props: IDetailsListProps): number {
-  const { cellStyleProps = DEFAULT_CELL_STYLE_PROPS } = props;
+function getPaddedWidth(column: IColumn, paddingOnly?: true): number {
+  const cellStyleProps = DEFAULT_CELL_STYLE_PROPS;
```

Comment:
hmm. Is this a change in behavior? It looks like the user was able to pass along `cellStyleProps` which would be considered below, but now that prop would be ignored.